config/v201/component_config/standardized/InternalCtrlr.json

```diff
@@ -770,6 +770,22 @@
           "minimum": 1,
           "default": "60",
           "type": "integer"
+      },
+      "AllowSecurityLevelZeroConnections": {
+          "variable_name": "AllowSecurityLevelZeroConnections",
+          "characteristics": {
+            "supportsMonitoring": false,
+            "dataType": "boolean"
+          },
+          "attributes": [
+              {
+                  "type": "Actual",
+                  "mutability": "ReadOnly"
+              }
+          ],
+          "description": "If enabled the transactions that were active before shutdown will be resumed, if possible",
```

Comment:
Description doesn't fit the config key, probably a copy/paste error

Reply:
Whoops you are right :sweat_smile: